### DISCUSSION TITLE
ETH 2.0 Staking

### DISCUSSION THREAD

**Discussion by q-TheDog** [LedgerHQ/ledger-live-desktop#3361] [Q&A] [answered]
I saw recently that Kraken was offering ETH staking, and just wanted to see if this was in the plans for Ledger Live.

**Comment by gre** [comment]
https://www.youtube.com/watch?v=poQWhl3fGJw might have gave some insights :)

**Comment by jdetychey** [comment, marked as answer]
Stay tuned:
https://www.youtube.com/watch?v=d7r9kJALR6M
